parametermanager/regional_samples/regional_parametermanager_test.go

```diff
@@ -0,0 +1,301 @@
+// Copyright 2025 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package regional_parametermanager
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"os"
+	"strings"
+	"testing"
+
+	parametermanager "cloud.google.com/go/parametermanager/apiv1"
+	parametermanagerpb "cloud.google.com/go/parametermanager/apiv1/parametermanagerpb"
+	secretmanager "cloud.google.com/go/secretmanager/apiv1"
+	secretmanagerpb "cloud.google.com/go/secretmanager/apiv1/secretmanagerpb"
+	"github.com/GoogleCloudPlatform/golang-samples/internal/testutil"
+	"github.com/gofrs/uuid"
+	"google.golang.org/api/option"
+	grpccodes "google.golang.org/grpc/codes"
+	grpcstatus "google.golang.org/grpc/status"
+)
+
+func testName(tb testing.TB) string {
+	tb.Helper()
+
+	u, err := uuid.NewV4()
+	if err != nil {
+		tb.Fatalf("testName: failed to generate uuid: %v", err)
+	}
+	return u.String()
+}
+
+func testLocation(tb testing.TB) string {
+	tb.Helper()
+
+	v := os.Getenv("GOLANG_REGIONAL_SAMPLES_LOCATION")
+	if v == "" {
+		tb.Skip("testIamUser: missing GOLANG_REGIONAL_SAMPLES_LOCATION")
+	}
+
+	return v
+}
+
+func testClient(tb testing.TB) (*parametermanager.Client, context.Context) {
```

Comment:
issue: remove this function. Each test needs to ensure that it closes the client.